sos/report/plugins/gluster.py

```diff
@@ -35,7 +35,7 @@ def wait_for_statedump(self, name_dir):
             statedumps_present = statedumps_present+1
             ret = -1
             while ret == -1:
-                last_line = file(
+                last_line = open(
```

Comment:
Actually, one small adjustment is needed here before merge.

As written, this leaves us with an out of scope file object that never gets closed. Just need to wrap this in a with clause, e.g.

```python
with open(path, 'r') as sfile:
    last_line = sfile.readlines()[-1]
    ret = string.count(last_line, 'DUMP_END_TIME')
```